src/text-editor.coffee

```diff
@@ -677,7 +677,7 @@ class TextEditor extends Model
   # this editor.
   shouldPromptToSave: ({windowCloseRequested}={}) ->
     if windowCloseRequested
-      @isModified()
+      false
```

Comment:
Exactly, so following the above reproduction steps would only reopen the untitled buffers when starting it via `atom /somepath`. 

I feel like this would be a nice byproduct, as it would allow users to have several untitled editors based on the project they're working on. I cannot think of any other drawback in not prompting to *save* when closing a window, 'cause serializing the state acts a nice safety net in case users want to restore their changes.

I think I should clearly state here that this will affect also editors with a path (i.e. non-untitled), although I think this is actually fine and doesn't change the thoughts I expressed above.

What do you think?